src/common/Config.ts

```diff
@@ -15,46 +15,31 @@ export default class Config {
     return require("../config/game.json");
   }
 
-  /**
-   * Returns all configured requirements for all units
-   */
-  public static getRequirements() {
-    return require("../config/units.json").units.requirements;
-  }
-
-  /**
-   * Returns mappings between unit-name <-> unitID
-   */
-  public static getMappings() {
-    return require("../config/units.json").mappings;
-  }
-
   /**
    * Returns a list of all buildings with their costs and cost-increase-factor per level
    */
-  public static getBuildings() {
-    return require("../config/units.json").units.buildings;
+  public static getBuildings(): IBuilding[] {
```

Comment:
Could use read-only attributes (getters), avoid static methods.

Reply:
I don't see a problem using static methods here as they only return the static json data. What would be the benefit of getters over static methods in this case?

Reply:
Instances are easier to replace (dependency inject) than static calls. And ES6 getters are cool :P